website/source/docs/configuration/builder.html.erb

```diff
@@ -0,0 +1,70 @@
+---
+layout: "docs"
+page_title: "Server Configuration"
+sidebar_current: "docs-configuration"
+description: |-
+  Vault server configuration reference.
+---
+
+<h1>Vault Configuration</h1>
+
+<nav class="tabs">
+  <ul>
+    <li>
+      <a href="/docs/configuration/builder.html" class="is-active">
+        Configuration Builder
+      </a>
+    </li>
+    <li>
+      <a href="/docs/configuration/index.html">
+        Manual Configuration
+      </a>
+    </li>
+  </ul>
+</nav>
+
+<p>
+  Choose from the options below (some are required) and download your
+  configuration file. Some variables may be sensitive, so we will give you
+  placeholders that you can replace after downloading.
+</p>
+
+<form id="configuration-builder">
+  <%= partial "builder/section_listener" %>
+  <%= partial "builder/section_storage" %>
+  <%= partial "builder/section_telemetry" %>
+  <%= partial "builder/section_seal" %>
+
+  <%= partial "builder/reveal_select_field", locals: {
+    label: "Vault Web UI",
+    name: "ui",
+    options: {
+      false: "Do not activate UI",
+      true: "Activate UI"
```

Comment:
Do we want Activate to be the default here since the option would be left out if they don't check, but also if they do check and then don't switch the <select>?